ebpf/symtab/elf/buildid.go

```diff
@@ -93,7 +93,7 @@ func (f *MMapedElfFile) GNUBuildID() (BuildID, error) {
 		return BuildID{}, fmt.Errorf(".note.gnu.build-id is not a GNU build-id : %s", hex.EncodeToString(data))
 	}
 	rawBuildID := data[16:]
-	if len(rawBuildID) != 20 && len(rawBuildID) != 8 { // 8 is xxhash, for example in Container-Optimized OS
+	if len(rawBuildID) < 8 {
```

Comment:
this is the change for fixing ".note.gnu.build-id has wrong size" when gnu build id is 8 bytes long